source/linux/Overview/Top_Level_Makefile.rst

```diff
@@ -104,17 +104,17 @@ makefile targets.
         * AM62x installer supports ``am62xx-evm`` and ``am62xx-lp-evm`` platforms. ``am62xx-evm`` is the default platform for the Toplevel Makefile. To build for ``am62xx-lp-evm``, pass ``PLATFORM=am62xx-lp-evm`` as argument to make.
         * No special arguments are needed to build for ``am62xxsip-evm`` in AM62xSIP Installer.
 
-.. ifconfig:: CONFIG_part_variant not in ('AM62X', 'AM64X', 'AM62PX')
+.. ifconfig:: CONFIG_part_variant not in ('AM62X', 'AM64X', 'AM62PX', 'AM335X', 'AM437X', 'AM65X', 'AM57X')
 
     .. code-block:: console
 
         host# sudo apt-get install build-essential autoconf automake bison flex libssl-dev bc u-boot-tools swig
 
-.. ifconfig:: CONFIG_part_variant in ('AM62PX', 'AM62X', 'AM64X')
+.. ifconfig:: CONFIG_part_variant in ('AM62PX', 'AM62X', 'AM64X', 'AM335X', 'AM437X', 'AM65X', 'AM57X')
 
     .. code-block:: console
 
-        host# sudo apt-get install build-essential autoconf automake bison flex libssl-dev bc u-boot-tools swig python3 python3-pip
+        host# sudo apt-get install build-essential autoconf automake bison flex libmpc* libssl-dev bc u-boot-tools swig python3 python3-pip
```

Comment:
Is there a particular version of libmpc you want to call out here? A glob match is good for short term but long term it makes it difficult to decipher what the actual dependency was.

Reply:
Latest version of the packages should be fine. And added all the actual dependencies.